mixer/adapter/svcctrl/handler_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package svcctrl
+
+import (
+	"testing"
+
+	"istio.io/istio/mixer/adapter/svcctrl/template/svcctrlreport"
+	"istio.io/istio/mixer/template/apikey"
+	"istio.io/istio/mixer/template/quota"
+)
+
+func TestHandlerInterface(t *testing.T) {
```

Comment:
you can inline these tests in the code itself, a la: `var _ apikey.Handler = (*handler)(nil)`. Let's do that instead of relying on this test.